cli/src/lockfiles/javascript.rs

```diff
@@ -20,7 +21,7 @@ impl Parseable for PackageLock {
 
     /// Parses `package-lock.json` files into a vec of packages
     fn parse(&self) -> ParseResult {
-        let parsed: Value = serde_json::from_str(&self.0)?;
+        let parsed: JsonValue = serde_json::from_str(&self.0)?;
 
         let into_descriptor = |(name, v): (String, &Value)| {
```

Comment:
```suggestion
        let into_descriptor = |(name, v): (String, &JsonValue)| {
```
Just guessing at this...

Reply:
Already better at Rust than I am! :smile:

Reply:
LOL...that will likely never be true...but I am learning a lot from you.